ksqldb-rocksdb-config-setter/src/main/java/io/confluent/ksql/rocksdb/KsqlBoundedMemoryRocksDBConfig.java

```diff
@@ -54,7 +56,18 @@ public class KsqlBoundedMemoryRocksDBConfig extends AbstractConfig {
   private static final double INDEX_FILTER_BLOCK_RATIO_DEFAULT = 0.0;
   private static final String INDEX_FILTER_BLOCK_RATIO_DOC =
       "Percentage of the RocksDB block cache to set aside for high-priority entries, i.e., "
-      + "index and filter blocks.";
+          + "index and filter blocks.";
+
+  public static final String COMPACTION_STYLE_CONFIG = CONFIG_PREFIX + "compaction.style";
+  private static final String COMPACTION_STYLE_DOC =
+      "All RocksDB instances across all KSQL servers will apply this compaction style "
+          + "to remove the invalidated records of the state store.";
+
+  public static final String COMPRESSION_TYPE_CONFIG = CONFIG_PREFIX + "compression.type";
```

Comment:
Just checking: do we know that having one compression setting for all SST files at all levels is acceptable for the use cases we're targeting? I'm all for starting simple if possible, just want to make sure we won't have to make an immediate follow-up change to allow more granular control if it turns out that's required for certain users.